pkgs/development/python-modules/accelerate/default.nix

```diff
@@ -113,6 +115,31 @@ buildPythonPackage rec {
       "test_init_trackers"
       "test_log"
       "test_log_with_tensor"
+
+      # After enabling MPS in pytorch, these tests started failing
```

Comment:
If they only fail on Darwin, they should only be disabled there.

Reply:
Yes, that's why they are in the block which checks for stdenv.hostPlatform.isDarwin

Reply:
using `++ lib.optionals stdenv.hostPlatform.isDarwin [ ... ];`.